app/mailers/order_assignment_mailer.rb

```diff
@@ -0,0 +1,19 @@
+class OrderAssignmentMailer < BaseMailer
+
+  def notify_assigned_user(order_details)
+    @order_details = Array(order_details)
+    return if @order_details.blank?
+
+    @user = @order_details.first.assigned_user
+    return if @user.blank?
```

Comment:
Yes, that's the unlikely scenario this is trying to avoid. Part of what makes it unlikely is I don't believe there's a way to unassign through the modal, only through batch assignment.